<?xml version="1.0" encoding="UTF-8" ?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:dc="http://purl.org/dc/elements/1.1/">
  <channel>
    <title>Redash instance went down automatically</title>
    <link>https://discuss.redash.io/t/redash-instance-went-down-automatically/10292</link>
    <description>### Issue Summary

Our redash instance was running fine and suddenly it went down, seeing lot of DB errors in logs, could someone share any inputs


ProgrammingError: (psycopg2.ProgrammingError) relation &quot;organizations&quot; does not exist
Tue, Apr 12 2022 8:16:55 am LINE 2: FROM organizations
Tue, Apr 12 2022 8:16:55 am ^
Tue, Apr 12 2022 8:16:55 am [SQL: &#39;SELECT organizations.updated_at AS organizations_updated_at, organizations.created_at AS organizations_created_at, organizations.id AS organizations_id, organizations.name AS organizations_name, organizations.slug AS organizations_slug, organizations.settings AS organizations_settings \nFROM organizations \nWHERE organizations.slug = %(slug_1)s \n LIMIT %(param_1)s&#39;] [parameters: {&#39;slug_1&#39;: &#39;default&#39;, &#39;param_1&#39;: 1}] (Background on this error at: http://sqlalche.me/e/f405)
Tue, Apr 12 2022 8:16:55 am [2022-04-12 13:16:55,780] ERROR in app: Exception on /favicon.ico [GET]

### Technical details:

* Redash Version: * Version: 8.0.2+b37747 (a9d7ca43)
* Browser/OS: Chrome
* How did you install Redash: github</description>
    
    <lastBuildDate>Tue, 12 Apr 2022 17:15:14 +0000</lastBuildDate>
    <category>Self Hosted Redash Support</category>
    <atom:link href="https://discuss.redash.io/t/redash-instance-went-down-automatically/10292.rss" rel="self" type="application/rss+xml" />
      <item>
        <title>Redash instance went down automatically</title>
        <dc:creator><![CDATA[Rash1d]]></dc:creator>
        <description><![CDATA[
            <p>thanks Jesse for your reply.<br>
Only thing I see is that the postgresql pod was started yesterday. why would it start by itself and we need to create the database every time the pod restarts.</p>
          <p><a href="https://discuss.redash.io/t/redash-instance-went-down-automatically/10292/3">Read full topic</a></p>
        ]]></description>
        <link>https://discuss.redash.io/t/redash-instance-went-down-automatically/10292/3</link>
        <pubDate>Tue, 12 Apr 2022 16:44:45 +0000</pubDate>
        <guid isPermaLink="false">discuss.redash.io-post-10292-3</guid>
        <source url="https://discuss.redash.io/t/redash-instance-went-down-automatically/10292.rss">Redash instance went down automatically</source>
      </item>
      <item>
        <title>Redash instance went down automatically</title>
        <dc:creator><![CDATA[jesse]]></dc:creator>
        <description><![CDATA[
            <p>That error means that the database schema doesn’t exist on the database that Redash connected to. <em>Normally</em>, we only see that error when you first set-up Redash. It happens if you start the application before running the <code>create_tables</code> script.</p>
<p>For a production instance there could be a lot of causes, and there isn’t enough information in your post to indicate why. It could be that the IP address or DNS record for the metadata database changed. It could be that the whole database was dropped (this would not happen automatically).</p>
          <p><a href="https://discuss.redash.io/t/redash-instance-went-down-automatically/10292/2">Read full topic</a></p>
        ]]></description>
        <link>https://discuss.redash.io/t/redash-instance-went-down-automatically/10292/2</link>
        <pubDate>Tue, 12 Apr 2022 16:22:28 +0000</pubDate>
        <guid isPermaLink="false">discuss.redash.io-post-10292-2</guid>
        <source url="https://discuss.redash.io/t/redash-instance-went-down-automatically/10292.rss">Redash instance went down automatically</source>
      </item>
      <item>
        <title>Redash instance went down automatically</title>
        <dc:creator><![CDATA[Rash1d]]></dc:creator>
        <description><![CDATA[
            <h3>
<a name="issue-summary-1" class="anchor" href="https://discuss.redash.io#issue-summary-1"></a>Issue Summary</h3>
<p>Our redash instance was running fine and suddenly it went down, seeing lot of DB errors in logs, could someone share any inputs</p>
<p>ProgrammingError: (psycopg2.ProgrammingError) relation “organizations” does not exist<br>
Tue, Apr 12 2022 8:16:55 am LINE 2: FROM organizations<br>
Tue, Apr 12 2022 8:16:55 am ^<br>
Tue, Apr 12 2022 8:16:55 am [SQL: ‘SELECT organizations.updated_at AS organizations_updated_at, organizations.created_at AS organizations_created_at, organizations.id AS organizations_id, organizations.name AS organizations_name, organizations.slug AS organizations_slug, organizations.settings AS organizations_settings \nFROM organizations \nWHERE organizations.slug = %(slug_1)s \n LIMIT %(param_1)s’] [parameters: {‘slug_1’: ‘default’, ‘param_1’: 1}] (Background on this error at: <a href="http://sqlalche.me/e/f405" class="inline-onebox" rel="noopener nofollow ugc">Error Messages — SQLAlchemy 1.4 Documentation</a>)<br>
Tue, Apr 12 2022 8:16:55 am [2022-04-12 13:16:55,780] ERROR in app: Exception on /favicon.ico [GET]</p>
<h3>
<a name="technical-details-2" class="anchor" href="https://discuss.redash.io#technical-details-2"></a>Technical details:</h3>
<ul>
<li>Redash Version: * Version: 8.0.2+b37747 (a9d7ca43)</li>
<li>Browser/OS: Chrome</li>
<li>How did you install Redash: github</li>
</ul>
          <p><a href="https://discuss.redash.io/t/redash-instance-went-down-automatically/10292/1">Read full topic</a></p>
        ]]></description>
        <link>https://discuss.redash.io/t/redash-instance-went-down-automatically/10292/1</link>
        <pubDate>Tue, 12 Apr 2022 13:22:55 +0000</pubDate>
        <guid isPermaLink="false">discuss.redash.io-post-10292-1</guid>
        <source url="https://discuss.redash.io/t/redash-instance-went-down-automatically/10292.rss">Redash instance went down automatically</source>
      </item>
  </channel>
</rss>
